loaders/src/glTF/2.0/Extensions/KHR_materials_sheen.ts

```diff
@@ -84,7 +84,15 @@ export class KHR_materials_sheen implements IGLTFLoaderExtension {
             babylonMaterial.sheen.roughness = 0;
         }
 
+        if (properties.sheenRoughnessTexture) {
+            promises.push(this._loader.loadTextureInfoAsync(`${context}/sheenRoughnessTexture`, properties.sheenRoughnessTexture, (texture) => {
+                texture.name = `${babylonMaterial.name} (Sheen Roughness)`;
+                babylonMaterial.sheen.textureRoughness = texture;
+            }));
+        }
+
         babylonMaterial.sheen.albedoScaling = true;
+        babylonMaterial.sheen.useRoughnessFromMainTexture = false;
```

Comment:
Should we check to see if the two textures can be combined and set this to true? Might have to check for KHR_texture_transform which will make this a bit complicated. Ditto for sheen.

Reply:
The check is done by the PBR code (sheen and clearcoat doing the same thing here): if both (internal) textures are identical, only one is passed to the shader and is used for both cases.

Reply:
Hrm, I looked for it and didn't see it. I must have missed it.

Reply:
Look for CLEARCOAT_TEXTURE_ROUGHNESS_IDENTICAL  and SHEEN_TEXTURE_ROUGHNESS_IDENTICAL